### PR TITLE
Adding stubs to PicardLiftovervcf, GATK4_mergevcfs and filterMutectCalls

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Comment by Lucpen** [comment]
There is an error related to Conda that I am unsure on how to fix or if it is necessary to fix it:
`Caused by:
  Failed to create Conda environment
  command: conda create --mkdir --yes --quiet --prefix /home/runner/pytest_workflow_3j9p2vgr/picard_liftovervcf_test_picard_liftovervcf_stubs/work/conda/env-aa7c3f17b3e9b256fbd55d1bfa076bd8 bioconda::picard=2.27.1
  status : 1
  message:
    PaddingError: Placeholder of length '80' too short in package /home/runner/pytest_workflow_3j9p2vgr/picard_liftovervcf_test_picard_liftovervcf_stubs/work/conda/env-aa7c3f17b3e9b256fbd55d1bfa076bd8/bin/Rscript.
    The package must be rebuilt with conda-build > 2.0.
`

File: modules/gatk4/filtermutectcalls/main.nf [hunk 1]
```diff
@@ -54,4 +54,15 @@ process GATK4_FILTERMUTECTCALLS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+    stub:
```
Comment (ramprasadn) [inline modules/gatk4/filtermutectcalls/main.nf:57]:
```suggestion

        stub:
```

File: modules/gatk4/filtermutectcalls/main.nf [hunk 2]
```diff
@@ -54,4 +54,15 @@ process GATK4_FILTERMUTECTCALLS {
...
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.vcf.gz
+    touch ${prefix}.vcf.gz.tbi
+    touch ${prefix}.vcf.gz.filteringStats.tsv
+    cat <<-END_VERSIONS > versions.yml
```
Comment (ramprasadn) [inline modules/gatk4/filtermutectcalls/main.nf:63]:
```suggestion

    cat <<-END_VERSIONS > versions.yml
```

File: modules/gatk4/mergevcfs/main.nf [hunk 3]
```diff
@@ -43,4 +43,13 @@ process GATK4_MERGEVCFS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.vcf.gz
+    cat <<-END_VERSIONS > versions.yml
```
Comment (ramprasadn) [inline modules/gatk4/mergevcfs/main.nf:50]:
```suggestion

    cat <<-END_VERSIONS > versions.yml
```

**Review by ramprasadn — COMMENTED** [review]
Nice work, Lucia! 

`-entry` parameters in the nextflow run command should point to the name of the workflow defined in main.nf file. Those need to be updated, and then a few minor suggestions around formatting that's all :) 👍🏻

I am not sure what's causing the conda testrun to fail, but hopefully it will go away as you fix the entry 🤞🏻

File: modules/gatk4/mergevcfs/main.nf [hunk 4]
```diff
@@ -43,4 +43,15 @@ process GATK4_MERGEVCFS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+
+    stub:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    touch ${prefix}.vcf.gz
+
```
Comment (FriederikeHanssen) [inline modules/gatk4/mergevcfs/main.nf:51]: @asp8200 added tbi recently, prob also needs to in the stub

**Review by muffato — APPROVED** [review]

**Comment by muffato** [comment]
@FriederikeHanssen @ramprasadn . The tests now pass, I think this is good for merging.

**Review by FriederikeHanssen — APPROVED** [review]

**Pull request merged**